code/_global_vars/lists/misc.dm

```diff
@@ -10,6 +10,8 @@ GLOBAL_LIST_EMPTY(tagger_locations)
 
 GLOBAL_LIST_INIT(char_directory_tags, list("Pred", "Pred-Pref", "Prey", "Prey-Pref", "Switch", "Non-Vore", "Unset"))
 GLOBAL_LIST_INIT(char_directory_erptags, list("Dominant", "Dom-Pref", "Submissive", "Sub-Pref", "Switch", "No ERP", "Unset"))
+GLOBAL_LIST_INIT(char_directory_sexualitytags, list("Straight", "Bisexual", "Pansexual", "Gay", "Lesbian", "Asexual", "Demisexual", "Unset"))
+GLOBAL_LIST_INIT(char_directory_gendertags, list("Male", "Female", "Nonbinary", "Trans Man", "Trans Woman", "Other", "Ungendered", "Unset"))
```

Comment:
Sexuality and gender may want to be re-reviewed by staff since this was a feature previous discussed and denied for an up-port from what I remember. If staff have already glanced at it then it's probably fine.

[Past Discussion context (virgo discord, #dev-general)](https://discord.com/channels/173831852602294272/187013248309002240/1041243568599212032)